IT/src/test/java/com/microsoft/gctoolkit/integration/aggregation/TestRuntimeAggregation.java

```diff
@@ -1,41 +1,27 @@
-package com.microsoft.gctoolkit.sample.aggregation;
+package com.microsoft.gctoolkit.integration.aggregation;
 
 import com.microsoft.gctoolkit.time.DateTimeStamp;
+import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-
-public class RuntimeAggregationTest {
-
-    private static class TestRuntimeAggregation extends RuntimeAggregation {
-
-        @Override
-        public boolean hasWarning() {
-            return false;
-        }
-
-        @Override
-        public boolean isEmpty() {
-            return false;
-        }
-    }
+public class TestRuntimeAggregation {
```

Comment:
Isn't this the wrong way around? The class name should  have Test as a suffix in order for surefire to pick it up.